runelite-client/src/main/java/net/runelite/client/plugins/clanchat/ClanChatPlugin.java

```diff
@@ -70,6 +106,64 @@ public void updateClanChatTitle()
 		}
 	}
 
+	private boolean send(ChatboxInput chatbox)
+	{
+		executor.execute(() ->
+		{
+			chatbox.resume();
+		});
+
+		return true;
+	}
+
+	private boolean isClanChatClosed()
+	{
+		Widget widget = client.getWidget(WidgetInfo.CLAN_CHAT);
+
+		if (widget == null)
+		{
+			return true;
+		}
+
+		return widget.isHidden();
```

Comment:
Can be simplified to `return widget == null || widget.isHidden();` and removing the null check above.